### PR TITLE
feat: progress bar when scanning

### PR DESCRIPTION
**Pull request by skymoore**
Add a progress bar when scanning multiple namespaces

pbar

### CODE REVIEW

**Comment by rothgar** [comment]
Thanks for the PR! This looks really good.

Could you also run `make THIRD_PARTY.txt` and commit that new file to this branch? With new dependencies I want to verify the licenses on the libraries and we need to keep it up to date when we distribute binaries.

Thanks

**Comment by skymoore** [comment]
@rothgar done :)

File: Makefile [hunk 1]
```diff
@@ -35,4 +35,4 @@ THIRD_PARTY.txt: licenses
 	mv licenses/THIRD_PARTY.txt ./
 
 licenses:
-	go-licenses save github.com/rothgar/dds --save_path ./licenses
```
Comment (rothgar) [inline Makefile:38]: how'd that get there? :laughing:

Comment (skymoore) [reply Makefile:38]: I can't imagine where it may have come from... complete mystery.

**Pull request merged**